lib/python/rosie/svn_post_commit.py

```diff
@@ -147,7 +147,8 @@ def run(self, repos, revision, no_notification=False):
         os.environ["TZ"] = "UTC"
         date_time_str = self._svnlook("date", "-r", revision, repos)
         date, dtime, _ = date_time_str.split(None, 2)
```

Comment:
Is `date_time_str` still a string?

`date` and `dtime` get used as bytes on the next line.

Reply:
Fair play. Variable name changed to reflect.